src/marlin_stubs/pause/pause.cpp

```diff
@@ -175,6 +137,28 @@ void plan_pause_e_move_notify_progress(const float &length, const feedRate_t &fr
     }
 }
 
+//todo move
```

Comment:
make a src/common/RAII.h and place it there

Reply:
fixed